decidim-core/app/views/pages/home/_highlighted_processes.html.erb

```diff
@@ -1,37 +1,5 @@
 <section class="wrapper-home">
-  <div class="row">
-    <h3 class="section-heading"><%= t(".active_processes") %></h3>
-    <div class="row collapse">
-      <div class="row small-up-1 smallmedium-up-2 mediumlarge-up-3
-                  large-up-4 card-grid">
-        <% highlighted_participatory_processes.each do |process| %>
-          <div class="column">
-            <article class="card card--process card--mini">
-              <%= link_to decidim_participatory_processes.participatory_process_path(process), class: "card__link" do %>
-                <div class="card__image-top"
-                     style="background-image:url(<%= process.hero_image.url %>)"></div>
-              <% end %>
-              <div class="card__content">
-                <%= link_to decidim_participatory_processes.participatory_process_path(process), class: "card__link" do %>
-                  <h4 class="card__title"><%= translated_attribute process.title %></h4>
-                <% end %>
-                <% if process.active_step %>
-                  <span class="card--process__small">
-                    <%= t(".active_step") %>
-                    <strong><%= translated_attribute process.active_step.title %></strong>
-                  </span>
-                <% end %>
-              </div>
-            </article>
-          </div>
-        <% end %>
-      </div>
-    </div>
-  </div>
-  <div class="row">
-    <div class="columns small-centered small-12
-                smallmedium-8 medium-6 large-4">
-      <%= link_to t(".see_all_processes"), decidim_participatory_processes.participatory_processes_path, class: "button expanded hollow button--sc home-section__cta" %>
-    </div>
-  </div>
+  <% Decidim.view_hooks.get(:highlighted_elements).each do |hook| %>
+    <%= render partial: hook[:partial] %>
```

Comment:
I wouldn't directly render a partial, as we don't have enough information about what's going on. Instead, I'd register a block and evaluate it here, inline:

```ruby
instance_eval hook[:block]
```

This way, the block could to stuff like:

```ruby
Decidim.view_hooks.register(:highlighted_elements, priority: LOW_PRIORITY) do
  participatory_processes = Decidim::ParticipatoryProcess.active
  render partial: "decidim/participatory_processes/home/list", collection: participatory_processes
  render partial: "decidim/participatory_processes/home/stats"
end
```

Reply:
Isn't that equivalent? 😕

Reply:
No, as this way you get the chance of having some kind of ruby code there as well. Think of it as it were a mini-controller and a mini-view. A cell, maybe?